md/templates/md/agency_detail.html

```diff
@@ -63,7 +63,45 @@ <h2 id="stop-percentage">Stops data</h2>
 </div>
 {% endblock stop-display %}
 
-{% block search-rate-display %}{% endblock search-rate-display %}
+{% block search-rate-display %}
+  <!-- Fraction of stops searched -->
+  <div class="row">
+     <div class="col-md-12">
+      <h3 id="search-percentage-dept">{% if officer_id %}Officer{% else %}Departmental{% endif %} search rate</h3>
+
+      <div role="tabpanel">
+
+        <!-- Nav tabs -->
+        <ul class="nav nav-tabs" role="tablist">
+          <li role="presentation" class="active">
+            <a href="#stop_search_line_div" role="tab" data-toggle="tab">Chart</a></li>
+          <li role="presentation">
+            <a href="#stop_search_data" role="tab" data-toggle="tab">Data</a></li>
+        </ul>
+
+        <!-- Tab panes -->
+        <div class="tab-content">
+          <div role="tabpanel" class="tab-pane active" id="stop_search_line_div">
+            <svg id="stop_search_line"></svg>
+          </div>
+          <div role="tabpanel" class="tab-pane" id="stop_search_data">
+          </div>
+        </div>
+
+      </div>
+
+      <p class="help-block">
+        This chart is a longitudinal representation of the average departmental
+        search rate for vehicle stops since the department began reporting its
+        data to the MD Department of Justice.  The black line represents the
+        overall search rate for all motorists.
+        <br><br>
+        Click the "Data" tab to review the actual stop/search counts from the
+        MD Department of Justice.
```

Comment:
I don't think this is the name of the agency in Maryland that gave us the stats.